code/game/objects/structures/ai_core.dm

```diff
@@ -144,7 +144,7 @@
 					user << "<span class='warning'>This MMI does not seem to fit!</span>"
 					return
 
-				if(M.hacked)
+				if(M.hacked || M.clockwork)

```

Comment:
What if you could set the MMI to run an antag conversion proc on the AI's core before being deleted?